### PR TITLE
Changes required for migration to new cluster

### PR DESCRIPTION
**Pull request by harshad16**
Changes required for migration to new cluster
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: #1971 

## Description

This PR contains following changes:
1. Rename of overlays from cnv-prod, zero-prod to moc-prod. 
2. Bucket prefixes are named as smaug-prod
3. Adjustment of the chat notifications.

### CODE REVIEW

File: core/overlays/moc-prod/backend-prod/thoth-notification.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ spec:
             - "-H"
             - "Content-Type: application/json; charset=UTF-8"
             - "-d"
-            - "{'text':'I have successfully synchronized *Thoth Core* components to *CNV-PROD* *infra* namespace, see <https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications/prod-thoth-core-infra|ArgoCD UI> 🚀'}"
+            - "{'text':'I have successfully synchronized *Thoth Core* components to *PROD* *backend* namespace, see <https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications/prod-thoth-core-backend|ArgoCD UI> 🚀'}"
             - "$(THOTH_DEVOPS_WEBHOOK_URL)"
```
Comment (Gregory-Pereira) [inline core/overlays/moc-prod/backend-prod/thoth-notification.yaml:23]: Why was this file not only migrated environments but also moved from infra to backend namespace?

File: docs/environments.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This section describes the different environments used to deploy all the Thoth S
 
 Project Thoth uses Postgresql database for knowlegde graph storage.
 
-- Deployment manifest can be found at: [Graph Manifest](../core/overlays/cnv-prod/graph-prod)
+- Deployment manifest can be found at: [Graph Manifest](../core/overlays/PROD/graph-prod)
```
Comment (Gregory-Pereira) [inline docs/environments.md:12]: This folder should be moc-prod

File: prow/add-ist [hunk 3]
```diff
@@ -56,7 +56,7 @@ def add_tag(imagestream: dict, tag_name: str) -> None:
 
 
 stream = None
-with open("overlays/cnv-prod/imagestreamtags.yaml") as ist:
+with open("overlays/PROD/imagestreamtags.yaml") as ist:
```
Comment (Gregory-Pereira) [inline prow/add-ist:59]: in the prow folder, i only see overlays/smaug in your branch, not overlays/PROD

File: prow/add-ist [hunk 4]
```diff
@@ -69,5 +69,5 @@ for data in stream:
 
     updated_imagestream_tags.append(data)
 
-with open("overlays/cnv-prod/imagestreamtags.yaml.new", "w") as ist:
+with open("overlays/PROD/imagestreamtags.yaml.new", "w") as ist:
```
Comment (Gregory-Pereira) [inline prow/add-ist:72]: same here

**Comment by Gregory-Pereira** [comment]
why is this folder named: thoth-application/slo-reporter/overlays/moc-prod-ocp4-stage/ instead of just moc-prod?

**Review by Gregory-Pereira — CHANGES_REQUESTED** [review]
I had questions about some of the folder names, not sure its wrong just confused on some of the naming conventions

File: core/overlays/moc-prod/backend-prod/thoth-notification.yaml (hunk 1 above)
Comment (harshad16) [reply core/overlays/moc-prod/backend-prod/thoth-notification.yaml:23]: That was an old issues, which was fixed in this.
As you can see the directory is named: **backend-prod**
The notification should relay it is syncing the **backend** namespace

**Comment by harshad16** [comment]
> why is this folder named: thoth-application/slo-reporter/overlays/moc-prod-ocp4-stage/ instead of just moc-prod?

The slo-reporter is executed in the PSI ocp4 environment because it sends an email in response, when an email arrives outside the red-hat VPN, the email services mark it as spam.
so we execute the production slo-reporter also in the PSI environment.

File: prow/add-ist (hunk 3 above)
Comment (harshad16) [reply prow/add-ist:59]: thanks, fixing it :+1:

File: prow/add-ist (hunk 4 above)
Comment (harshad16) [reply prow/add-ist:72]: thanks,fixing it :+1:

**Comment by harshad16** [comment]
@Gregory-Pereira  Thanks for the review
Fixed the suggested issues
Ready for review again.

**Comment by harshad16** [comment]
Merging it manually
thanks for reviewing.

**Pull request merged**